proto/cosmos/pubkey/v1beta1/tx.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package cosmos.pubkey.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/crypto/secp256k1/keys.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/auth/pubkey/types";
+
+// MsgChangePubKey defines a message that enables changing a pubkey
+// of an account.
+message MsgChangePubKey {
+  bytes address = 1 [
+    (gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"
+  ];
+  cosmos.crypto.secp256k1.PubKey pub_key = 2;
```

Comment:
Should not assume that this is a secp256k1 key.  Should use generic pubkey interface instead.

https://github.com/sikkatech/cosmos-sdk/blob/4a44f5aa3ef57aa08011dbab9f263ee2fbcfe53e/proto/cosmos/auth/v1beta1/auth.proto#L18